lib/iris/tests/test_merge.py

```diff
@@ -25,7 +25,7 @@
 import iris.tests.stock
 
 
-class TestMixin:
+class MergeMixin:
```

Comment:
`pytest` tried to run this as a test class, so errored all three tests when it couldn't find `_data_path`.  Removing "Test" from the class name stops `pytest` trying to do that.